policyengine_us/variables/gov/states/ca/dhcs/ffyp/ca_ffyp_eligible.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class ca_ffyp_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible person for California Former Foster Youth Program"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dhcs.ca.gov/services/medi-cal/eligibility/Pages/FFY_Bene.aspx"
+    defined_for = StateCode.CA
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.ca.dhcs.ffyp
+        age = person("age", period)
+        age_eligible = p.foster_care_age_minimum <= age < p.age_limit
```

Comment:
Does the person have to be in foster care in the prior year?

Could you add comments to the logic explaining the eligibility pieces

Reply:
- not exactly clear but technically since it says "former foster" I just changed it so that `age > foster_care_age_minimum` instead of `>=`
- I don't know how to write a test case for this and couldn't find other examples either
- how do I integrate this with other variables

Reply:
> Does the person have to be in foster care in the prior year?
> 
> Could you add comments to the logic explaining the eligibility pieces

- not exactly clear but technically since it says "former foster" I just changed it so that `age > foster_care_age_minimum` instead of `>=`
- I don't know how to write a test case for this and couldn't find other examples either
- how do I integrate this with other variables